Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -4,7 +4,7 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
 . $PSScriptRoot/Common/SonarQubeHelpers/SonarQubeHelper.ps1
 
-if (IsPrBuild -and (GetTaskContextVariable "DisableSQAnalysisOnPrBuilds" -eq "true")) 
+if ( (IsPrBuild) -and ((GetTaskContextVariable "DisableSQAnalysisOnPrBuilds") -eq "true"))
```

Comment:
I added paranthesis because otherwise the rest of the tokens would be considered arguments to IsPrBuild ... oh powershell...